sandbox/gvisor/run.py

```diff
@@ -48,7 +48,6 @@
 sys.stderr.flush()
 
 include_bash = args.command == 'bash'
-include_python2 = args.command == 'python2'
 include_python3 = args.command == 'python3'
```

Comment:
This variable is not used. Should we delete it, delete from the arguments and adapt the caller? 
As an alternative, we may replace this line with `assert args.command == 'python3'`.